web/pdf_history.js

```diff
@@ -23,10 +23,11 @@
   } else {
     factory((root.pdfjsWebPDFHistory = {}));
```

Comment:
Don't you need to import `domEvents` here, similar to web/pdf_link_service.js just below, since it's always going to be undefined otherwise?